docs/event-service/1_about.md

```diff
@@ -0,0 +1,13 @@
+---
+sidebar_position: 1
+---
+
+# About
+
+The event service allows users to schedule and manage both online and offline events with comprehensive details. The service is versatile, supporting various event types and providing users with the ability to create and customize events with specific attributes and preferences.
+
+For offline events, users can provide details such as the physical location, latitude, and longitude.For online events, users can SET any created meeting links like zoom , google Meet by passing the relevant details in the onlineDetails object. in the onlineDetails object.
```

Comment:
fix typos
- ". in the onlineDetails object." seems repeat
- use lowercase for SET